ui/next.config.js

```diff
@@ -2,4 +2,55 @@ const withTM = require('next-transpile-modules')(['domain-logic'])
 
 module.exports = withTM({
   experimental: { esmExternals: true },
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: securityHeaders,
+      },
+    ]
+  },
 })
+
+// The declaration below will allow Youtube Videos and Google Fonts
+const ContentSecurityPolicy = `
+  default-src 'self';
+  script-src 'self' 'unsafe-eval' 'unsafe-inline' *.youtube.com;
+  child-src *.youtube.com;
+  style-src 'self' 'unsafe-inline' *.googleapis.com;
+  img-src * blob: data:;
+  media-src 'none';
+  connect-src *;
```

Comment:
having a * here seems too broad. 
Usually we just need to connect to the main API (which we could fetch from the environment) and a few thrid party providers (which the developer could manually specify).

Reply:
Done